app/views/admin/members/edit.html.haml

```diff
@@ -123,7 +123,7 @@
                         %span.fa.fa-trash
 
 
-                - unless ENV['MAILCHIMP_DATACENTER'].blank?
+                - unless ENV['MAILCHIMP_DATACENTER'].blank? || @member.user.nil?
```

Comment:
The error seems to happen because mailchimp_interests returns nil.
This happens when the email address isn't on the mailing list.
Instead the mailchimp interests could be hidden if `@member.mailchimp_interests.nil?`